Forms/Tab.php

```diff
@@ -115,7 +122,10 @@ public function getTabSet() {
 	}
 
 	public function extraClass() {
-		return implode(' ', (array)$this->extraClasses);
+		$classes = (array)$this->extraClasses;
+		$classes[] = 'tab-pane';
```

Comment:
This adds bootstrap classes to every tab use (even outside of the CMS). It's a shame this can't be implemented through a new `Includes/Tab.ss` template, which we could override with a CMS-specific template - because a tab is rendered through `<div $AttributesHTML>`. `FormField->AttributesHTML()` doesn't allow customisation of the attributes, so we can't merge PHP and template specified classes. And even if we implement that in the `FormField` API, the template parser doesn't support non-string arguments like `<div $AttributesHTML(["class" => "tab-pane"])>` (see `SSTemplateParser.inc.php` `Argument_Lookup()`). That'll be a pain to implement. 

@flamerohr Can you please document the reason this class is on `Tab`?
@tractorcow Do you think this is a worthwhile tradeoff for now?